p1.ipynb

```diff
@@ -35,7 +35,7 @@
   },
```

Comment:
<p>prueba de comentario, qué buen histograma, usted es un genio la verdad</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TheCsarbeat/proyecto1-IA/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>hola</p>

Reply:
<p>Test of comment by ReviewNB</p>